### DISCUSSION TITLE
Display one hemisphere

### DISCUSSION THREAD

**Discussion by tourette95** [simnibs/simnibs#295] [9. Results and analysis]
Hi everyone!

I was wondering if there is a way to show only one cerebral hemisphere or the medial parts. Likewise it would be great to be able to visualize the brain separated by hemispheres, especially for tDCS protocols that use both cerebral hemispheres.

Any ideas?

Thanks in advance!

:D!

ps: I mean: is there any way to make cuts to the brain in the results window?

**Comment by jduemose** [comment]
Hi I think [this](https://simnibs.github.io/simnibs/build/html/tutorial/visualization.html) tutorial might be useful. I think you should be able to show one hemisphere at a time using the "visibility" option. There is also a tool for clipping the surfaces if you want to make arbitrary cuts.

Best,
Jesper

**Reply by tourette95** [reply]
The Clipping feature sounds promissing, I will try it!
Thanks! 
:)!

**Reply by tourette95** [reply]
Update:
Indeed I was able to achieve the desired results using the Clipping tool, thank you very much!
